### PR TITLE
Fixes DEVELOPER-3380 (topic pages in drupal)

### PR DESCRIPTION
**Pull request by LightGuard**
Please see https://issues.jboss.org/browse/DEVELOPER-3380 for more
information.

* Fixed composer to use official package location
* Added new twig functions to use in the front end
* Updated styles in topics to prevent side scrolling
* Updated topic layout to send the correct layout for drupal.

### CODE REVIEW

**Comment by LightGuard** [comment]
@adelasofia @robpblake please check

**Comment by robpblake** [comment]
retest this please

File: _docker/drupal/drupal-filesystem/web/modules/custom/redhat_developers/src/Twig/RedHatDevelopersTagsExtension.php [hunk 1]
```diff
@@ -7,23 +7,25 @@
 
 namespace Drupal\redhat_developers\Twig;
```
Comment (robpblake) [inline _docker/drupal/drupal-filesystem/web/modules/custom/redhat_developers/src/Twig/RedHatDevelopersTagsExtension.php]: Can this be tested with PHPUnit or other recommended PHP unit testing framework?

**Comment by robpblake** [comment]
@LightGuard This looks fine as far as I can tell. Probably better to wait for @adelasofia to take a look as she probably knows the original issues better than I.

cheers,

Rob

**Comment by adelasofia** [comment]
@LightGuard Looks great! 

The only problem I found were the broken links in the grey box in:
"Technologies" section in http://developers-pr.stage.redhat.com/pr/1348/build/1916/devops/ 
and 
"Development teams need to " section in http://developers-pr.stage.redhat.com/pr/1348/build/1916/enterprise-java/

File: _docker/drupal/drupal-filesystem/web/modules/custom/redhat_developers/src/Twig/RedHatDevelopersTagsExtension.php (hunk 1 above)
Comment (LightGuard) [inline _docker/drupal/drupal-filesystem/web/modules/custom/redhat_developers/src/Twig/RedHatDevelopersTagsExtension.php]: Yeah, though I'm not sure how to set it up. I know @drupalbee is working on that. I can give it a go, though I'm not quite sure how we hook it up into the current pipeline.

**Comment by LightGuard** [comment]
@adelasofia I shouldn't have touched those. Are they broken in prod?

File: _docker/drupal/drupal-filesystem/web/modules/custom/redhat_developers/src/Twig/RedHatDevelopersTagsExtension.php (hunk 1 above)
Comment (drupalbee) [inline _docker/drupal/drupal-filesystem/web/modules/custom/redhat_developers/src/Twig/RedHatDevelopersTagsExtension.php]: yes, I am still working on this as well as the auto enable module request.

**Comment by adelasofia** [comment]
@LightGuard You're right, they are broken on prod too. I've created an issue DEVELOPER-3387

**Comment by LightGuard** [comment]
@adelasofia if you're happy with it, please merge.

**Pull request merged**